src/index.js

```diff
@@ -8,8 +8,8 @@ const Toasted = {
         }
 
         const Toast = new T(options);
-        Vue.component('toasted', ToastComponent);
-        Vue.toasted = Vue.prototype.$toasted = Toast;
+        app.component('toasted', ToastComponent);
+        app.config.globalProperties.$toasted = Toast;
```

Comment:
where is `app` coming from? I doubt that it is a global variable.

Reply:
L5:  fix argument `Vue -> app `

Reply:
@morr You need to rename `Vue` to `app` on the `install` function or revert to the previous name.

```suggestion
        Vue.component('toasted', ToastComponent);
        Vue.config.globalProperties.$toasted = Toast;
```